recipes/stringzilla/all/test_package/test_package_3_0.cpp

```diff
@@ -12,7 +12,7 @@ int main(void) {
   auto substring_position = haystack.find(needle); // Or `rfind`
 
   haystack.end() - haystack.begin() == haystack.size(); // Or `rbegin`, `rend`
-  haystack.find_first_of(" \w\t") == 4; // Or `find_last_of`, `find_first_not_of`, `find_last_not_of`
+  haystack.find_first_of(" \v\t") == 4; // Or `find_last_of`, `find_first_not_of`, `find_last_not_of`
```

Comment:
Any insight into why this change was needed? Thanks!

Reply:
This is modified in upstream since https://github.com/ashvardanian/StringZilla/pull/171.

Reply:
Thanks!